Cargo.toml

```diff
@@ -132,6 +132,7 @@ iota-crypto = { git = "https://github.com/iotaledger/crypto.rs", rev = "47460d64
     "slip10",
     "ed25519",
     "secp256k1",
+    "x25519",
```

Comment:
Maybe add "zeroize"` too (they other flags might have activated it already, but does not hurt)